src/components/entity.jsx

```diff
@@ -0,0 +1,126 @@
+import DocumentTitle from 'react-document-title'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import config from '../config'
+import { entityShape } from '../openfisca-proptypes'
+
+
+class Entity extends React.Component {
+  static propTypes = {
+    countryPackageName: PropTypes.string.isRequired,
+    countryPackageVersion: PropTypes.string.isRequired,
+    intl: intlShape.isRequired,
+    entityId: PropTypes.string.isRequired,
+    entity: entityShape.isRequired,
+  };
+
+  
+  render() {
+    const { entityId, entity } = this.props
+
+    return (
+      <DocumentTitle title={entityId + ' — ' + this.props.intl.formatMessage({ id: 'appName' })}>
+        <section>
+          <h1><code>{ entityId }</code></h1>
+          {
+            entity.plural
+            ? <span>
+                <h5><code>{ entity.plural }</code></h5>
+              </span>
+            : <em><FormattedMessage id="noPlural"/></em>
+          }
+          { entity.description
+            ? <p className="description">{ entity.description }</p>
+            : <em><FormattedMessage id="noDescription"/></em>
+          }
+          { this.renderEntityMetadata(entity) }
+          {
+            entity.documentation &&
+            <div>
+              <span className="message"><FormattedMessage id="documentation"/></span>
+              <span className="message">{ entity.documentation }</span>
+            </div>
+          }
+          <hr/>
+          <a rel="external" target="_blank" href={`${config.apiBaseUrl}/entities`}>
+            <FormattedMessage id="rawJSONData"/>
+          </a>
+        </section>
+      </DocumentTitle>
+    )
+  }
+
+  renderEntityRole = (roleId, role) => {
+    return (
+      <tr key={ roleId }>
+        <td>{ roleId }</td>
+        <td>
+          { 
+            role.plural
+            ? <span className="message">{ role.plural }</span>
+            : <em><FormattedMessage id="noPlural"/></em>
+          }
+        </td>
+        <td>
+          { 
+            role.description
+            ? <span className="message">{ role.description }</span>
+            : <em><FormattedMessage id="noDescription"/></em>
+          }
+        </td>
+        <td>
+          { 
+            role.max 
+            ? <span className="message">{ role.max }</span> 
+            : <em><FormattedMessage id="noRoleMaxPersons"/></em>
+          }
+        </td>
+      </tr>
+    )
+  }
+
+  renderEntityMetadata = (entity) => {
```

Comment:
Seems like this method renders the role descriptions rather than the metadata.
What about renaming it to `renderRoles` and take `roles` as a parameter?